riscv_ctg/data/imc.yaml

```diff
@@ -1846,11 +1849,13 @@ c.jr:
   isa: 
     - IC
   formattype: 'crformat'
+  rs1_val_data: 'gen_sign_dataset(xlen)+ gen_sp_dataset(xlen)'
+  rs2_val_data: 'gen_sign_dataset(xlen)+ gen_sp_dataset(xlen)'
   template: |-
 
     // $comment
-    // opcode: c.jr; op1:$rs1
-    TEST_CJR_OP($testreg, $rs1, $swreg, $offset)
+    // opcode: c.jr; op1:$rs1; op2:$rs2; op1val:$rs1_val; op2val:$rs2_val
+    TEST_CJR_OP($testreg, $rs1, $rs2, $swreg, $offset)
```

Comment:
This change does not look correct. The template does not match the macro definition in the `arch_test.h` file.

Reply:
Reverted the change since the template did not require the change.